tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/openwire/FQQNOpenWireTest.java

```diff
@@ -315,13 +314,13 @@ public void testVirtualTopicFQQN() throws Exception {
    }
 
    @Test
-   @Ignore("need to figure auto bindings creation")
    public void testVirtualTopicFQQNAutoCreate() throws Exception {
       Connection exConn = null;
 
       SimpleString topic = new SimpleString("VirtualTopic.Orders");
       SimpleString subscriptionQ = new SimpleString("Consumer.A");
 
+      // defaults are false via test setUp
```

Comment:
We should test the case where an address already exists and the default type is ANYCAST.

Reply:
thanks @mtaylor  - I see, that getDefaultAddressRoutingType should only be used by auto address creation. The queue routing type needs to be figured out from the address.